docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -140,46 +143,114 @@ If compiler auto-detection fails for some reason, specify it using
+On Level 2-4 systems, users will need to modify certain environment variables, such as the path to HPC-Stack, so that the SRW App can find and load the appropriate modules. For systems with Lmod installed, one of the current ``build_<platform>_<compiler>`` modulefiles can be copied and used as a template. To check whether Lmod is installed, run ``echo $LMOD_PKG``, and see if it outputs a path to the Lmod package. On systems without Lmod, users can modify or set the required environment variables with the ``export`` or ``setenv`` commands despending on whether they are using a bash or csh/tcsh shell, respectively: 
 
 .. code-block::
 
    export <VARIABLE_NAME>=<PATH_TO_MODULE>
    setenv <VARIABLE_NAME> <PATH_TO_MODULE>
 
+.. _MacDetails:
+
+Additional Details for Building on MacOS
+------------------------------------------
+
+.. note::
+    Users not building the SRW App to run on MacOS may skip to the :ref:`next section <BuildExecutables>`. 
+
+The SRW App can be built on MacOS systems, presuming HPC-Stack has already been successfully installed. The following two options have been tested:
+
+* **Option 1:** MacBookAir 2020, M1 chip (arm64, running natively), 4+4 cores, Big Sur 11.6.4, GNU compiler suite v.11.2.0_3 (gcc, gfortran, g++); no MPI pre-installed
+
+* **Option 2:** MacBook Pro 2015, 2.8 GHz Quad-Core Intel Core i7 (x86_64), Catalina OS X 10.15.7, GNU compiler suite v.11.2.0_3 (gcc, gfortran, g++); no MPI pre-installed
+
+.. note::
+    Examples in this subsection presume that the user is running Terminal.app with a bash shell environment. If this is not the case, users will need to adjust the commands to fit their command line application and shell environment. 
+
+The ``build_macos_gnu`` modulefile initializes the module environment, lists the location of HPC-Stack modules, loads the meta-modules and modules, and sets compilers, additional flags, and environment variables needed for building the SRW App. The modulefile must be modified to include the absolute path to the user's HPC-Stack installation and ``ufs-srweather-app`` directories. In particular, the following section must be modified:
+
+.. code-block:: console
+
+   # This path should point to your HPCstack installation directory
+   setenv HPCstack "/Users/username/hpc-stack/install"
+
+   # This path should point to your SRW Application directory
+   setenv SRW "/Users/username/ufs-srweather-app"
+   
```

Comment:
One of the motivation for using modulefiles is to set the environment irrespective of the login shell used (bash or tcsh).
      
      module load build_macos_gnu

should work for bash even though it is written in "csh-like" syntax. The only place where login shell dependent code is needed is when loading Lmod itself i.e.
     
      source etc/lmod-setup.sh macos
                 or
      source etc/lmod-setup.csh macos

Reply:
@natalie-perlin This code is already in the modulefile, which is written in csh-like syntax. In reality, the user is only changing the path. If we change this to bash, we have to redo the entire modulefile, which I don't think is worth it since, as @danielabdi-noaa pointed out, the modulefile should work regardless of the user's shell.